language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/CommandClient.java

```diff
@@ -28,6 +37,13 @@ Single<Empty> submitAndWait(
       @NonNull Optional<Duration> deduplicationTime,
       @NonNull List<@NonNull ? extends HasCommands> commands);
 
+  /**
+   * This method has been deprecated as of [???]. Please use {@link
```

Comment:
Reminder to go through this file and replace the `[???]` placeholders with `@since`. As for the version, it depends on when it gets merged. I would mark it as 2.5 and if by any chance this ends up being merged after 2.5 I would go back and fix the version.

Reply:
A mentioned by @S11001001, `@since` is not meant to be used like this. Apologies for the confusion. I think you can use the `since` field of [the `@Deprecated` annotation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Deprecated.html) (notice the difference in capitalization)